LiteCore/Support/FilePath.cc

```diff
@@ -124,7 +124,7 @@ static int copyfile(const char* from, const char* to)
     err = CopyFile2(wideFrom, wideTo, nullptr);
 #endif
 
-    if(err != S_OK) {
+    if(err < S_OK) {
```

Comment:
I kind of understand but this seems like only half of the needed change.  CopyFileW should not be used anymore because we don't support Windows 7 anyway so that is some small debt that needs to be paid back by removing the dynamic search for CopyFile2.  I think most error codes here are greater than 0 so this needs either a removal of CopyFileW (recommended) or both values need to be coalesced to the same value.